src/components/FormLinks/FormLinks.tsx

```diff
@@ -5,34 +5,63 @@ import './FormLinks.css'
 /**
  * @typedef IFormLinksProps
  * @props {string} [className] - the CSS classes
+ * @props {string} [path] - the name of route path
  */
 
 export interface IFormLinksProps {
   className?: string
+  path?: string
 }
 
 /**
  * Renders form component with links
  *
  * @param  {string} [className] - the CSS classes
+ * @param {string} [path] - the name of route path
  *
  * @example
- * <FormLinks className='a6y-react-auth-form' />
+ * <FormLinks path='sign-in' className='a6y-react-auth-form' />
  */
 
 const FormLinks = ({
   className = 'a6y-react-auth__form',
+  path,
 }: IFormLinksProps): JSX.Element => {
-  return (
-    <div className={className + '__links'}>
-      <Link className={className + '__link'} to='/forgot-password'>
-        Forgot Password
-      </Link>
-      <Link className={className + '__link'} to='/sign-up'>
-        Sign Up
-      </Link>
-    </div>
-  )
+  function renderLinks(): JSX.Element {
+    switch (path) {
+      case 'sign-in':
+        return (
+          <>
+            <Link className={className + '__link'} to='/forgot-password'>
```

Comment:
Here we have a code repetition. Convert links like `Forgot password` to a consts, ie:

```typescript
const ForgotPasswordLink = () => {
  return (
    <Link className={className + '__link'} to='/forgot-password'>
      Forgot Password
    </Link>
  )
}

// ...and then use it in the renderLinks
```